setup.py

```diff
@@ -31,7 +31,10 @@
 
 tensorboard_extra_require = ["tensorflow >=2.3.0, <=2.5.0"]
 metadata_extra_require = ["pandas >= 1.0.0"]
-full_extra_require = tensorboard_extra_require + metadata_extra_require
+xai_extra_require = ["tensorflow-cpu>=2.3.0, <=2.5.0"]
+full_extra_require = (
```

Comment:
tensorboard and xai have the same dependencies so we may need to deduplicate this with `list(set(...))`.